api/api.go

```diff
@@ -29,51 +29,22 @@ type npmPackageResponse struct {
 }
 
 type NpmPackageVersion struct {
-	Name         string            `json:"name"`
-	Version      string            `json:"version"`
-	Dependencies map[string]string `json:"dependencies"`
+	Name         string                        `json:"name"`
+	Version      string                        `json:"version"`
+	Dependencies map[string]*NpmPackageVersion `json:"dependencies"`
 }
 
 func packageHandler(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	pkgName := vars["package"]
 	pkgVersion := vars["version"]
```

Comment:
[review] [nit] could drop these vars and just use them in the calling locations if you want since there's only one call per now